ProfessorAssessor/View Controllers/LoginViewController.m

```diff
@@ -0,0 +1,50 @@
+#import "LoginViewController.h"
+#import "Parse/Parse.h"
+#import "User.h"
+#import "School.h"
+#import "Professor.h"
+#import "Course.h"
+#import "Review.h"
+#import "Networker.h"
+
+@interface LoginViewController ()
+
+@property (strong, nonatomic) IBOutlet UITextField *username;
+@property (strong, nonatomic) IBOutlet UITextField *password;
+
+- (IBAction)didTapLogin:(UIButton *)sender;
+- (IBAction)didTapSignUp:(UIButton *)sender;
+
+@end
+
+@implementation LoginViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+}
+
+- (IBAction)didTapLogin:(UIButton *)sender {
+    NSString *username = self.username.text;
+    NSString *password = self.password.text;
+    // TODO: check if username and password are valid before method call
+    [PFUser logInWithUsernameInBackground:username password:password block:^(PFUser * user, NSError *  error) {
+        if (error != nil) {
+            // TODO: handle error
+        } else {
+            // TODO: manually segue to logged in view
+            [self performSegueWithIdentifier:@"loginSegue" sender:self];
+        }
+    }];
+}
+
+- (IBAction)didTapSignUp:(UIButton *)sender {
+    // TODO: make new user and set username and password
+
+    // TODO: set school after selected from table view
```

Comment:
You'll want to have a different view for this I'm guessing. I would expect a segue to a reusable SchoolSelectionViewController